scripts/setup_gae.sh

```diff
@@ -31,4 +31,22 @@ export PYTHONPATH=.:$COVERAGE_HOME:$GOOGLE_APP_ENGINE_HOME:$GOOGLE_APP_ENGINE_HO
 # Delete old *.pyc files
 find . -iname "*.pyc" -exec rm -f {} \;
 
+# Moved from install .travis.yml.
```

Comment:
Oh, I think I see -- the code was in two places and you consolidated it. That's great! But I don't think you need the comment.